redaxo/src/addons/cronjob/pages/system.log.cronjob.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// this file integrates the already existing log-viewer as a syslog page.
+// the required registration wiring can be found in the package.yml
+require __DIR__. '/log.php';
```

Comment:
Man könnte die Datei auch weglassen, und in der package.yml `subPath: pages/log.php` setzen.
Aber keine Ahnung, ob das besser ist, so wie hier ist es vielleicht klarer.
Mir egal ;)

Reply:
Ich habe bewusst es hier genauso gemacht wie in den anderen log integrationen.